shared/naturalcrit/markdown.js

```diff
@@ -295,37 +295,125 @@ const superSubScripts = {
+						currentDefinition = {};
+					}
+					currentDefinition = {
+						dt  : this.lexer.inlineTokens(match[1].trim()),
+						dd  : [],
+						ddo : []
+					};
+				} else if(_.isEmpty(currentDefinition)) {
+					return;
+				}
+				// Test for a DD value.
+				if(match[2].trim().length>0) {
+					if((match[1]?.length > 0) && (match[0].indexOf('\n') > match[1]?.length)) { // Inline Style DD
+						currentDefinition.dd = currentDefinition.dd.concat([match[2].trim()]);
+						currentDefinition.dd.forEach((dd)=>{
+							currentDefinition.ddo.push(this.lexer.inlineTokens(dd));
+						});
+						allDefinitions.push(currentDefinition);
+						inlineDefinitions = true;
+						currentDefinition = {};
+						continue;
+					}
+					// Multi-line style DDs
+					const newDefinitions = _.flatten(match[2].split('\n::').filter((item)=>item).map((s)=>s.trim()));
+					if(newDefinitions?.length) {
+						currentDefinition.dd = currentDefinition.dd.concat(newDefinitions);
+					}
+				} else {
+					currentDefinition.dd.push('');
+				}
+				lastEmpty = false;
+			} else if(inList) { // Regular line that might mark the end of a line.
+				appending = false;
+				if(inlineDefinitions) {
+					endIndex -= match[0].length;
+					break;
+				}
+				if(match[0].trim().length == 0) {
+					if(lastEmpty) {
+						continue;
+					} else {
+						lastEmpty = true;
+					}
+				} else {
+					lastEmpty = false;
+					const lastPos = typeof currentDefinition.dd.length !== 'undefined' ? currentDefinition.dd.length - 1 : 0;
+					currentDefinition.dd[lastPos] = `${currentDefinition.dd[lastPos]} ${match[0].trim()}`;
+					appending = true;
+				}
+			}
+		}
+		if(currentDefinition.hasOwnProperty('dt')) {
+			currentDefinition.dd.forEach((dd)=>{
+				currentDefinition.ddo.push(this.lexer.inlineTokens(dd));
 			});
-			endIndex = regex.lastIndex;
+			allDefinitions.push(currentDefinition);
 		}
-		if(definitions.length) {
+		if(allDefinitions.length) {
 			return {
-				type : 'definitionLists',
-				raw  : src.slice(0, endIndex),
-				definitions
+				type        : 'definitionLists',
+				raw         : src.slice(0, endIndex),
+				inlineDefinitions,
+				definitions : allDefinitions
 			};
 		}
 	},
 	renderer(token) {
-		return `<dl>${token.definitions.reduce((html, def)=>{
-			return `${html}<dt>${this.parser.parseInline(def.dt)}</dt>`
-			            + `<dd>${this.parser.parseInline(def.dd)}</dd>\n`;
-		}, '')}</dl>`;
+		let returnVal = `<dl${token.inlineDefinitions ? ' class="inlineDL"' : ''}>`;
+		token.definitions.forEach((def)=>{
+			let dds = def.ddo.map((s)=>{
+				return `<dd>${this.parser.parseInline(s).trim()}</dd>`;
+			}).join('\n');
+			returnVal += `<dt>${this.parser.parseInline(def.dt)}</dt>${dds.indexOf('\n') > -1 ? '\n' : ''}${dds}\n`;
+		});
+		returnVal = returnVal.trim();
+		return `${returnVal}</dl>`;
 	}
 };
 
+Marked.use({ extensions: [mustacheSpans, mustacheDivs, mustacheInjectInline, definitionLists, superSubScripts] });
```

Comment:
You have two separate `Marked.use` blocks, with mostly the same extensions listed. Probably remove this one and add your `definitionLists` to the one below instead.

Reply:
Bad merge there. All cleaned up.